### ISSUE TITLE
Byte array being mapped as any.

### ISSUE THREAD

**Issue by beldrew** [vojtechhabarta/typescript-generator#462]
It's stated that byte should map to number, however the following java class the byte array gets mapped to any.  Is there a jackson2 annotation I'm missing?

``` java
public class BrowserFileItem implements BrowserFileItemInterface {
    private final Long id;
    private final byte[] data;
    private final String fileName;
    private final String fileExtension;
    private final String mimeType;

    public BrowserFileItem(final Long id, final byte[] data, final String fileName, final String fileExtension, final String mimeType) {
        this.id = id;
        this.data = data;
        this.fileName = fileName;
        this.fileExtension = fileExtension;
        this.mimeType = mimeType;
    }

    public Long getId() {
        return this.id;
    }

    public byte[] getData() {
        return this.data != null ? this.data.clone() : null;
    }

    public String getFileName() {
        return this.fileName;
    }

    public String getFileExtension() {
        return this.fileExtension;
    }

    public String getMimeType() {
        return this.mimeType;
    }
}
```

Becomes

``` typescript
    interface BrowserFileItemInterface {
        id: number;
        fileName: string;
        data: any;
        mimeType: string;
        fileExtension: string;
    }
```

**Comment by vojtechhabarta** [comment]
By default Jackson sends byte array as Base64 string. For example result of
``` java
public class BrowserFileItem {
    public byte[] data;
}
public static void main(String[] args) throws Exception {
    BrowserFileItem inst = new BrowserFileItem();
    inst.data = new byte[]{65, 66, 67};
    System.out.println(new ObjectMapper().writeValueAsString(inst));
}
```
is following JSON
``` json
{
    "data": "QUJD"
}
```

Unfortunately it is not possible to use `customTypeMappings` parameter to change mapping of primitive types and arrays but maybe this could be improved.

**Comment by vojtechhabarta** [comment]
Released in [2.30.840](https://github.com/vojtechhabarta/typescript-generator/releases/tag/v2.30.840).
